doc/guide/dynamics/dynamics-monte.rst

```diff
@@ -203,110 +339,111 @@ For instance, consider the following T1 simulation of a qubit with a lifetime of
     plt.show()
 
 
-The original sampling algorithm samples the no-jump trajectory on average 96.7% of the time, while the improved
-sampling algorithm only does so once.
+The original sampling algorithm samples the no-jump trajectory on average 96.7%
+of the time, while the improved sampling algorithm only does so once.
 
 
-.. _monte-reuse:
+.. _monte-seeds:
 
-Reusing Hamiltonian Data
-------------------------
+Reproducibility
+---------------
 
-.. note:: This section covers a specialized topic and may be skipped if you are new to QuTiP.
+For reproducibility of Monte-Carlo computations it is possible to set the seed
 
+.. code-block::
 
-In order to solve a given simulation as fast as possible, the solvers in QuTiP take the given input operators and break them down into simpler components before passing them on to the ODE solvers.
-Although these operations are reasonably fast, the time spent organizing data can become appreciable when repeatedly solving a system over, for example, many different initial conditions.
-In cases such as this, the Monte Carlo Solver may be reused after the initial configuration, thus speeding up calculations.
+    >>> res1 = mcsolve(H, psi0, tlist, c_ops, e_ops=e_ops, seeds=1, ntraj=1)
+    >>> res2 = mcsolve(H, psi0, tlist, c_ops, e_ops=e_ops, seeds=1, ntraj=1)
+    >>> res3 = mcsolve(H, psi0, tlist, c_ops, e_ops=e_ops, seeds=2, ntraj=1)
+    >>> np.allclose(res1, res2)
+    True
+    >>> np.allclose(res1, res3)
+    False
 
+The ``seeds`` parameter can either be an integer or numpy SeedSequence, which
+will then be used to create seeds for each trajectories. Or a list of
+interger/SeedSequence with one seed for each trajectories. Seeds available in
+the result object can be used to redo the same evolution:
```

Comment:
```suggestion
The ``seeds`` parameter can either be an integer or a numpy ``SeedSequence``, which
will then be used to create seeds for each trajectory. Alternatively it may be a list of
intergers or ``SeedSequence``s with one seed for each trajectories. Seeds available in
the result object can be used to redo the same evolution:
```